src/index.html

```diff
@@ -22,6 +22,397 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="containerHeader">
+      <a
+        href="#"
+        class="containerHeader__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo Moyo"
+        />
+      </a>
+      <nav class="navigation">
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a class="navigation__link is-active">Apple</a>
```

Comment:
The 'is-active' class should be applied to an anchor tag that has an 'href' attribute to indicate that it is the current page. Without 'href', the 'is-active' state might not make sense semantically.